src/OrchardCore.Modules/OrchardCore.Widgets/Views/ContentCard-WidgetsListPart.Edit.cshtml

```diff
@@ -8,18 +8,18 @@
     var widgetContentTypes = ContentDefinitionManager.ListTypeDefinitions().Where(t => t.GetStereotype() == "Widget");
 }
 
-<div class="widget widget-editor card my-1 @(Model.Inline != true ? "collapsed" : "")">
-    <div class="widget-editor-header card-header text-muted py-1 ps-3 pe-1">
-        @if (Model.CanMove != false)
+<div class="widget widget-editor card@(Model.Inline ? " collapsed" : String.Empty)">
+    <div class="widget-editor-header card-header text-muted py-1 ps-2 pe-1">
+        @if (Model.CanMove)
         {
             <div class="widget-editor-handle"><i class="fas fa-arrows-alt" aria-hidden="true"></i></div>
         }
         <button type="button" class="btn btn-outline-secondary btn-sm widget-list-editor-btn-toggle widget-editor-btn-collapse"><i class="fa fa-angle-down" aria-hidden="true"></i></button>
         <button type="button" class="btn btn-outline-secondary btn-sm widget-list-editor-btn-toggle widget-editor-btn-expand"><i class="fa fa-angle-@(Orchard.CultureDir() == LanguageDirection.LTR ? "right" : "left")" aria-hidden="true"></i></button>
         @contentType
-        <div class="btn-widget-metadata border border-info w-100">
+        <div class="btn-widget-metadata border border-info w-100 my-2">
             <div class="btn-group">
-                @if (Model.CanInsert != false)
+                @if (Model.CanInsert)
```

Comment:
This looks like it's broken the View (in some cases, not sure which ones yet)
But `Model.CanInsert` is genearlly a null, so it can't be cast to a bool

Reply:
If that's the case then it should be set properly as a bool beforehand in the Driver or we should be using a strongly type ShapeViewModel.